llama_stack/apis/agents/openai_responses.py

```diff
@@ -17,6 +17,28 @@ class OpenAIResponseError(BaseModel):
     message: str
 
 
+@json_schema_type
+class OpenAIResponseInputMessageContentText(BaseModel):
+    text: str
+    type: Literal["input_text"] = "input_text"
+
+
+@json_schema_type
+class OpenAIResponseInputMessageContentImage(BaseModel):
+    detail: Literal["low"] | Literal["high"] | Literal["auto"] = "auto"
+    type: Literal["input_image"] = "input_image"
+    # TODO: handle file_id
+    image_url: str | None = None
+
+
+# TODO: handle file content types
```

Comment:
nit: also audio?

Reply:
To this general point, one of the next things I'd like to do is stub in the entire Responses API surface even if it's not all implemented yet.

However, on this specific point, the Responses API does not yet support audio. That's only in Chat Completion API for now, although I expect it will get added to Responses at some point. But, for now, only image, text ,and file content types are supported in Responses.